resolwe_bio/tests/processes/test_support_processors.py

```diff
@@ -548,3 +548,33 @@ def test_umi_tools_dedup(self):
 
         dedup_paired = self.run_process('umi-tools-dedup', {'alignment': bam_paired.id})
         self.assertFile(dedup_paired, 'stats', './corall/output/dedup_paired_stats.txt')
+
+    @tag_process('seqtk-rev-complement-single', 'seqtk-rev-complement-paired')
+    def test_reverse_complement(self):
+        with self.preparation_stage():
+            reads = self.run_process('upload-fastq-single', {
+                'src': ['hs_slamseq_R1.fastq.gz']
+            })
+
+            paired_reads = self.prepare_paired_reads(['hs_slamseq_R1.fastq.gz'],
+                                                     ['hs_slamseq_R2.fastq.gz'])
+
+            inputs_single = {
+                'reads': reads.id
+            }
+
+            revcomp_single = self.run_process('seqtk-rev-complement-single', inputs_single)
+
+            self.assertFiles(revcomp_single, 'fastq', ['hs_slamseq_R1_complemented.fastq.gz'],
+                             compression='gzip')
+
+            inputs_paired = {
+                'reads' : paired_reads.id,
+                'select_mate' : 'Mate 1'
+            }
+
+            revcomp_paired = self.run_process('seqtk-rev-complement-paired', inputs_paired)
+            self.assertFiles(revcomp_paired, 'fastq', ['hs_slamseq_R1_complemented.fastq.gz'],
```

Comment:
This line can fit into 119 chars limit, which is the limit for this repo. It is more compact and easier to read if it is just in one line IMO. But this is a matter of style, so i leave it up to you.